### ISSUE TITLE
Backend Error when inserting multiple posts

### ISSUE THREAD

**Issue by khanh-nguyen** [khanh-nguyen/node-blogger-cli#1]
```
Error: Backend Error
    at Request._callback (node_modules/googleapis/lib/transporters.js:83:15)
    at Request.self.callback (node_modules/googleapis/node_modules/request/request.js:373:22)
    at Request.emit (events.js:110:17)
    at Request.<anonymous> (node_modules/googleapis/node_modules/request/request.js:1318:14)
    at Request.emit (events.js:129:20)
    at IncomingMessage.<anonymous> (node_modules/googleapis/node_modules/request/request.js:1266:12)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:907:16
    at process._tickCallback (node.js:372:11)
```

**Comment by khanh-nguyen** [comment]
May need to setTimeout to wait between inserting. Simply setTimeout inside `_.map` won't work.

**Comment by khanh-nguyen** [comment]
Fixed in PR#4.
